decidim-admin/app/commands/decidim/admin/update_component.rb

```diff
@@ -3,18 +3,17 @@
 module Decidim
   module Admin
     # This command gets called when a component is created from the admin panel.
-    class UpdateComponent < Decidim::Command
+    class UpdateComponent < Decidim::Commands::UpdateResource
```

Comment:
Maybe it doesn't make much sense to inherit from UpdateResource here, as it isn't doing much. Or the idea is that absolutely all the command classes that Create/Update/Delete should inherit from these always?